hi-ml-cpath/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -566,3 +573,119 @@ def test_init_weights_options(pretrained_encoder: bool, pretrained_pooling: bool
             assert mock_copy_weights.call_count == sum(
                 [int(pretrained_encoder), int(pretrained_pooling), int(pretrained_classifier)]
             )
+
+
+def _get_base_deepmil_module(
+    pretrained_encoder: bool = True,
+    pretrained_pooling: bool = True,
+    pretrained_classifier: bool = True,
+    n_classes: int = 3,
+    num_layers: int = 2,
+    num_heads: int = 1,
+    hidden_dim: int = 8,
+) -> BaseDeepMILModule:
+    module = BaseDeepMILModule(
```

Comment:
Wonder if it would be better to test this with TilesDeepMILModule - just aware it contains more params of the base class and it's what we're using now for experiments

Reply:
It doesn't matter much because all we're testing is defined in the Base Class but sure, we can change it to TilesDeepMILModule